.github/workflows/build-release.yml

```diff
@@ -184,3 +184,39 @@ jobs:
 
           cd ../installation
           ./bin/python3 -m test -uall
+
+  build-android:
+    needs:
+      - verify-input
+
+    # Android binary releases began in Python 3.14.
+    if: |
+       !(
+          startsWith(needs.verify-input.outputs.cpython_release, '3.9.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.10.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.11.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.12.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.13.')
+       )
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        arch: [aarch64, x86_64]
+        include:
+          - arch: aarch64
+            runs-on: macos-15
+          - arch: x86_64
+            runs-on: ubuntu-24.04
+
+    runs-on: ${{ matrix.runs-on }}
+    steps:
+      - name: "Checkout ${{ env.GIT_REMOTE }}/cpython"
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+          repository: "${{ env.GIT_REMOTE }}/cpython"
+          ref: "v${{ env.CPYTHON_RELEASE }}"
+
+      - uses: ./.github/actions/build-android
```

Comment:
Note: pending merge of https://github.com/python/cpython/pull/137186, let's set this as a draft for now.